ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml

```diff
@@ -172,6 +204,11 @@ OnboardingBasePage {
             anchors.topMargin: 24
             enabled: false
             property int gridCount: (grid.count === 20) ? 18 : grid.count
+
+            function checkMnemonicLength() {
+                submitButton.enabled = (root.mnemonicInput.length === (grid.atXBeginning ? 12 : submitButton.gridCount));
```

Comment:
I'm new to this code, so this is a question, that I just came up with.
Why do we set `submitButton.enabled` functionally instead of binding it to required properties?

Reply:
Good question. I just copied the previous code so I'm not 100% sure. My guess would be that since we are binded to an array (mnemonicInput), since the array itself doesn't change when pushing values to it, the change signal wouldn't be triggered.